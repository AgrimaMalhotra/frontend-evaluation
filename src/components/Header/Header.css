@import url('https://fonts.googleapis.com/css2?family=Big+Shoulders+Inline+Display:wght@500;600;700&display=swap');

.header > h1{
  margin: 0px; 
  font-family: 'Big Shoulders Inline Display', cursive;
  font-size: 70px;
}

.header{
  padding-top: 20px;
  padding-bottom: 20px;
  background-color: black;
  color: white;
  width:100vw;
max-height:100px;
}
