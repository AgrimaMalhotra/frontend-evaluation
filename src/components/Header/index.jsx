import React from 'react';
import './Header.css';
const Header = () => {
  return (
    <div className="header padding">
      <h1>EVENTIFY</h1>
    </div>
  );
};

export default Header;
