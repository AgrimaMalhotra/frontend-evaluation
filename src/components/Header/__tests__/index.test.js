import React from 'react';
import Header from '..';
import { render } from '@testing-library/react';

describe('Header', () =>
  it('should render correctly', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  }));
