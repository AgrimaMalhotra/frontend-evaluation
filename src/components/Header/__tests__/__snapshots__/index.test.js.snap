// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<div>
  <div
    class="header padding"
  >
    <h1>
      EVENTIFY
    </h1>
  </div>
</div>
`;
